src/services/category.js

```
@@ -73,6 +73,40 @@ const categoryService = {
     }
   
     return category;
+  },
+
+  getCategoryNames: async ({ search = '', page = 1, limit = 20 } = {}) => {
+    const query = {};
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)